docs/madr/decisions/009-tracing-policy.md

```diff
@@ -0,0 +1,184 @@
+# Traffic tracing policy
+
+- Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/4732
+
+## Context and Problem Statement
+
+We want to create a [new policy matching compliant](https://github.com/kumahq/kuma/blob/22c157d4adac7f518b1b49939c7e9ea4d2a1876c/docs/madr/decisions/005-policy-matching.md)
+resource for managing traffic tracing.
+
+## Decision Drivers
+
+- Replace existing policies with new policy matching compliant resources
+- [Path/Method filtering on traces](https://github.com/kumahq/kuma/issues/3335)
+- [Custom tags](https://github.com/kumahq/kuma/issues/3275)
+- Not blocking [OpenTelemetry tracing](https://github.com/kumahq/kuma/issues/3690)
+
+## Considered Options
+
+The new resource will be called `Tracing` in the rest of this document.
+
+Tracing in Kuma is implemented with Envoy's [tracing
+support](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing).
+At the moment, Kuma supports Zipkin and Datadog.
+
+### Naming
+
+Some potential names:
+
+- `Tracing`
+- `Traces`
+- `MeshTrafficTrace`
+
+#### Backends
+
+Currently, tracing support requires defining _backends_ in the `Mesh` resource.
+A backend defines where traces are sent, the _provider_. For `TrafficTrace`, it also contains the sampling rates settings.
+The `TrafficTrace` resource references these backends and contains no configuration of its own.
+
+The proposed `Tracing` policy follows the decisions about backends as laid out in
+[the MADR for `MeshAccessLog`](docs/madr/decisions/009-tracing-policy.md#backends). This
+means creating a new `TracingBackend` resource and allowing inline backend
+definitions.
+
+However, we propose moving configuring sampling and tag configuration
+from the backend (the `TrafficTrace` status quo) to the `Tracing` resource.
+Backends can contain the same settings but they can be set or overriden in
+specific `Tracing` resources.
+
+#### `targetRef`
+
+This is a new policy so it's based on `targetRef`. Envoy tracing is configured
+on the HTTP connection manager so `Tracing` has a single `spec.targetRef` field
+that selects what it applies to. It does not use `to` or `from` fields.
+
+All logging configuration happens under `spec.default` so that users are able to
+override settings with more specific `targetRef`s.
+
+Resources supported by `spec.targetRef` are `Mesh`, `MeshSubset`, `MeshService`,
+`MeshServiceSubset`, `MeshGatewayRoute` and eventually the future
```

Comment:
why is `MeshGatewayRoute` supported and `MeshHTTPRoute` not? I'm guessing there is a HCM per `MeshGatewayRoute` - right?

Reply:
Is `MeshHTTPRoute` the working name for `MeshTrafficRoute` or something else? If it's the same, I mention it in the next sentence.